### ISSUE TITLE
README statement about Python version in base environment is inaccurate

### ISSUE THREAD

**Issue by millsks** [conda-forge/miniforge#622]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

README states "Latest installers with Python 3.12 (*) in the base environment", but still using Python 3.10.  The base version of Python is updated in v24.5.0-0, but it is still listed as a pre-release.  I realize this is on hold until https://github.com/conda-forge/miniforge/issues/614 is resolved, but should the README be updated to state that the base environment is still being shipped with Python 3.10? Or are we expecting #614 to be resolved sooner rather than later?

https://github.com/conda-forge/miniforge/blob/09dd7a66373071c431ed3d8422d844e28af000db/Miniforge3/construct.yaml#L28-L32

![image](https://github.com/user-attachments/assets/0a503bde-4d4b-4417-871a-dc7c769cea23)

### Installed packages

```shell
# packages in environment at C:\Users\millsks\UserLocal\miniforge\py312_24300:
#
# Name                    Version                   Build  Channel
archspec                  0.2.3              pyhd8ed1ab_0    defaults
boltons                   24.0.0             pyhd8ed1ab_0    defaults
brotli-python             1.1.0           py310h00ffb61_1    defaults
bzip2                     1.0.8                hcfcfb64_5    defaults
ca-certificates           2024.2.2             h56e8100_0    defaults
certifi                   2024.2.2           pyhd8ed1ab_0    defaults
cffi                      1.16.0          py310h8d17308_0    defaults
charset-normalizer        3.3.2              pyhd8ed1ab_0    defaults
colorama                  0.4.6              pyhd8ed1ab_0    defaults
conda                     24.3.0          py310h5588dad_0    defaults
conda-libmamba-solver     24.1.0             pyhd8ed1ab_0    defaults
conda-package-handling    2.2.0              pyh38be061_0    defaults
conda-package-streaming   0.9.0              pyhd8ed1ab_0    defaults
distro                    1.9.0              pyhd8ed1ab_0    defaults
fmt                       10.2.1               h181d51b_0    defaults
idna                      3.6                pyhd8ed1ab_0    defaults
jsonpatch                 1.33               pyhd8ed1ab_0    defaults
jsonpointer               2.4             py310h5588dad_3    defaults
krb5                      1.21.2               heb0366b_0    defaults
libarchive                3.7.2                h313118b_1    defaults
libcurl                   8.7.1                hd5e4a3a_0    defaults
libffi                    3.4.2                h8ffe710_5    defaults
libiconv                  1.17                 hcfcfb64_2    defaults
libmamba                  1.5.8                h3f09ed1_0    defaults
libmambapy                1.5.8           py310h04f2035_0    defaults
libsolv                   0.7.28               h12be248_2    defaults
libsqlite                 3.45.2               hcfcfb64_0    defaults
libssh2                   1.11.0               h7dfc565_0    defaults
libxml2                   2.12.6               hc3477c8_1    defaults
libzlib                   1.2.13               hcfcfb64_5    defaults
lz4-c                     1.9.4                hcfcfb64_0    defaults
lzo                       2.10              he774522_1000    defaults
mamba                     1.5.8           py310hd9d798f_0    defaults
menuinst                  2.0.2           py310h00ffb61_0    defaults
miniforge_console_shortcut 1.0                  h57928b3_0    defaults
openssl                   3.2.1                hcfcfb64_1    defaults
packaging                 24.0               pyhd8ed1ab_0    defaults
pip                       24.0               pyhd8ed1ab_0    defaults
platformdirs              4.2.0              pyhd8ed1ab_0    defaults
pluggy                    1.4.0              pyhd8ed1ab_0    defaults
pybind11-abi              4                    hd8ed1ab_3    defaults
pycosat                   0.6.6           py310h8d17308_0    defaults
pycparser                 2.22               pyhd8ed1ab_0    defaults
pysocks                   1.7.1              pyh0701188_6    defaults
python                    3.10.14         h4de0772_0_cpython    defaults
python_abi                3.10                    4_cp310    defaults
reproc                    14.2.4.post0         hcfcfb64_1    defaults
reproc-cpp                14.2.4.post0         h63175ca_1    defaults
requests                  2.31.0             pyhd8ed1ab_0    defaults
ruamel.yaml               0.18.6          py310h8d17308_0    defaults
ruamel.yaml.clib          0.2.8           py310h8d17308_0    defaults
setuptools                69.5.1             pyhd8ed1ab_0    defaults
tk                        8.6.13               h5226925_1    defaults
tqdm                      4.66.2             pyhd8ed1ab_0    defaults
truststore                0.8.0              pyhd8ed1ab_0    defaults
tzdata                    2024a                h0c530f3_0    defaults
ucrt                      10.0.22621.0         h57928b3_0    defaults
urllib3                   2.2.1              pyhd8ed1ab_0    defaults
vc                        14.3                hcf57466_18    defaults
vc14_runtime              14.38.33130         h82b7239_18    defaults
vs2015_runtime            14.38.33130         hcb4865c_18    defaults
wheel                     0.43.0             pyhd8ed1ab_1    defaults
win_inet_pton             1.1.0              pyhd8ed1ab_6    defaults
xz                        5.2.6                h8d14728_0    defaults
yaml-cpp                  0.8.0                h63175ca_0    defaults
zstandard                 0.22.0          py310h0009e47_0    defaults
zstd                      1.5.5                h12be248_0    defaults
```

### Environment info

```shell
active environment : base
    active env location : C:\Users\millsks\UserLocal\miniforge\py312_24300
            shell level : 1
       user config file : C:\Users\millsks\.condarc
 populated config files : C:\Users\millsks\UserLocal\miniforge\py312_24300\.condarc
                          C:\Users\millsks\.condarc
          conda version : 24.3.0
    conda-build version : not installed
         python version : 3.10.14.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=skylake
                          __conda=24.3.0=0
                          __cuda=12.4=0
                          __win=0=0
       base environment : C:\Users\millsks\UserLocal\miniforge\py312_24300  (writable)
      conda av data dir : C:\Users\millsks\UserLocal\miniforge\py312_24300\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/selfexplainml/win-64
                          https://conda.anaconda.org/selfexplainml/noarch
          package cache : C:\Users\millsks\UserLocal\miniforge\py312_24300\pkgs
                          C:\Users\millsks\.conda\pkgs
                          C:\Users\millsks\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\millsks\UserLocal\miniforge\py312_24300\envs
                          C:\Users\millsks\.conda\envs
                          C:\Users\millsks\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/24.3.0 requests/2.31.0 CPython/3.10.14 Windows/10 Windows/10.0.22631 solver/libmamba conda-libmamba-solver/24.1.0 libmambapy/1.5.8
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by millsks** [comment]
Opened up a small PR with the update to the README, but if it is not needed go ahead and close it.

**Comment by hmaarrfk** [comment]
He release is currently blocked. Sorry. We are trying to fix it and hopefully release 3.12 build soon
